src/main.rs

```diff
@@ -66,15 +67,19 @@ fn get_workspace_windows(monitor: &str) -> Result<Vec<WorkspaceCustom>> {
             HyprError::NotOkDispatch("No active monitor found".to_string())
         })?
         .name;
+    let on_screen_workspaces: Vec<i32> = Monitors::get()?
+        .into_iter()
+        .map(|m| m.active_workspace.id).collect();
     let mut out_workspaces: Vec<WorkspaceCustom> = Vec::new();
 
     for workspace in workspaces
         .iter()
         .filter(|m| m.monitor == monitor || monitor == "ALL" || monitor == "_")
     {
         let mut active = false;
-        let mut class = format!("workspace-button w{}", workspace.id);
-        if active_workspace_id == workspace.id
+        let on_screen = on_screen_workspaces.contains(&workspace.id);
+        let mut class = format!("workspace-button w{}{}", workspace.id, if on_screen {" on-screen"} else {""});
```

Comment:
Can you update this to be `workspace-on-screen` to align with the other classes?